modules/countdown.js

```diff
@@ -3,37 +3,70 @@ require('moment-countdown')
 
 let timeout = null
 
-const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+function isPast (date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  return expiryDate.isBefore(now)
+}
+
+function nextBrexitDate () {
+  const article50 = '2020-01-31T00:00:00+01:00'
+  const transition = '2020-12-31T00:00:00+01:00'
+
+  if (isPast(transition)) {
+    return { name: null, date: null }
+  }
+  if (isPast(article50)) {
+    return { name: 'The transition period', date: transition }
+  }
+  return { name: 'Article 50', date: article50 }
+}
 
 function autoCount (bot, lastTick) {
-  const thisTick = moment(ARTICLE_50).countdown().toString().split(/, | and /)[0]
+  const eventName = nextBrexitDate.name
```

Comment:
You could do something cool with [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```suggestion
  const {name, date}  = nextBrexitDate
```